environments/ithc/ithc.tfvars

```diff
@@ -18,8 +18,8 @@ shutter_apps = [
   "TODO"
 ]
 
-cft_apps_ag_ip_address = "10.10.40.123"
-cft_apps_cluster_ips   = ["10.10.35.250", "10.10.39.250"]
+cft_apps_ag_ip_address = "10.11.225.123"
+cft_apps_cluster_ips   = ["10.11.192.250", "10.11.208.250"]
```

Comment:
Can you change these to last range in each subnet?
10.11.207
10.11.223

Reply:
now changed.